src/app/home/home.component.html

```diff
@@ -1,38 +1,25 @@
 <main class="main-div">
-    <section class="main-section-1">
-        <div class="card">
-            <div class="card-header">
-                Fazer publicação
-            </div>
-            <div class="card-body">
-                <div class="mb-3">
-                    <input class="form-control" type="text" placeholder="O que está cozinhando hoje?"
-                        aria-label="default input example">
-                    <textarea class="form-control" id="exampleFormControlTextarea1" rows="3"
-                        placeholder="Deixe um comentário sobre a receita."></textarea>
-                    <input class="form-control" type="file" id="formFile">
-                </div>
-                <a href="#" class="btn btn-primary">Publicar</a>
+
+    <div class="title-block">
+        <h1 id="title-page">Find meals by your ingredient</h1>
+        <p id="frase">A recipe has no soul. You, as the cook, must bring soul to the recipe.<br>-Thomas Keller</p>
+    </div>
+
+
+    <section class="search-section">
+            <div class="search-block">
```

Comment:
Eu usaria essa div como `form`, assim se algum usuário quisesse pesquisar por receitas apenas apertando "Enter", também seria possível.

Tem uma explicação sobre formulários no Angular: https://angular.io/start/start-forms#create-the-checkout-form

Não precisa alterar isso nesse PR necessariamente, porque assim terá mais tempo para fazer novas coisas no projeto, mas só para entender como funciona.

Reply:
Nossa que legal, não sabia, vou fazer dessa forma então. Valeu de novo kkk